src/components/Content/common/buttons.tsx

```diff
@@ -0,0 +1,172 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import { RippleButton } from 'components/Ripple';
+
+export const CreateButton = () => (
+  <RippleButton
+    basic
+    size="tiny"
+    css={(theme) => css`
+      &.ui.button.basic {
+        box-shadow: none;
+        color: ${theme.colors.primary_7} !important;
+        border: 1px solid ${theme.colors.primary_7};
+      }
+    `}
+    // disabled={isSaving}
+    // onClick={() => history.push(`/${resource.name.plural}/create`)}
+  >
+    Create
+  </RippleButton>
+);
+
+export const EditButton = () => (
+  <RippleButton
+    css={(theme) => css`
+      &.ui.button {
+        color: ${theme.colors.white};
+        background-color: ${theme.colors.accent};
+        border: 1px solid ${theme.colors.accent};
+        &:hover {
+          background-color: ${theme.colors.accent_dark};
+          border: 1px solid ${theme.colors.accent_dark};
+        }
+      }
+    `}
+    // disabled={isSaving}
+    // onClick={() => history.push(`/${resource.name.plural}/${id}/edit`)}
+    size="tiny"
+  >
+    Edit
+  </RippleButton>
+);
+
+export const DisableButton = () => (
+  <RippleButton
+    basic
+    size="tiny"
+    css={(theme) => css`
+      &.ui.button.basic {
+        box-shadow: none;
+        color: ${theme.colors.error_dark} !important;
+        border: 1px solid ${theme.colors.error_dark};
+      }
+    `}
+    // disabled={contentState === ContentState.DISABLING || get(item, 'status') === 'DISABLED'}
+    // loading={contentState === ContentState.DISABLING}
+    // onClick={async () => {
+    //   // direct DISABLE option available only for entities that cannot be deleted, currently just for USERS
+    //   if (resource.noDelete) {
+    //     setContentState(ContentState.DISABLING);
+    //     // update entity immediately, this action does not follow the stageChange/saveChange flow
+    //     const updated: Entity = await resource.updateItem({
+    //       item: { ...item, ...{ status: 'DISABLED' } },
+    //     });
+    //     await setItem(updated.id, resource);
+    //     await updateList(resource, parent);
+    //     setContentState(ContentState.DISPLAYING);
+    //   }
+    // }}
+  >
+    Disable
+  </RippleButton>
+);
+
+export const ConfirmDeleteButton = () => (
+  <RippleButton
```

Comment:
these buttons are self contained in components.
i think it's cleaner to use the useTheme() hook and just put the color into the css`` call

Reply:
or even use "styled" like you have used in other places